src/system/device.js

```diff
@@ -4,6 +4,10 @@
  * http://www.melonjs.org
  *
  */
+
+/*global wx*/
+/*eslint no-undef: "error"*/
```

Comment:
global should be declare here : 
https://github.com/melonjs/melonJS/blob/master/.eslintrc.json#L9

and please avoid adding more aslant exception

Reply:
OK, I follow, change later in PR.